Makefile

```diff
@@ -42,48 +50,62 @@ patch-dev: superset-clean
 	ln -s "$(PWD)/$(PATCH_SOURCE_DIR)/superset/superset_config_dev.py" "$(SUPERSET_DIR)/superset_config.py"; \
 
 # Create docker image
-.PHONY: superset-build
-superset-build: patch
-	docker build -t $(SUPERSET_IMAGE_NAME):$(VERSION) -f superset/contrib/docker/Dockerfile superset
+.PHONY: patch
+build: patch
+	docker build -t $(DOCKER_IMAGE_NAME):$(VERSION) -f superset/contrib/docker/Dockerfile $(SUPERSET_DIR)
+
+.PHONY: docker-validate
+docker-validate:
```

Comment:
Isn't this a copy-paste from [src-d/ci/Makefile::docker-validate](https://github.com/src-d/ci/blob/v1/Makefile.main#L183) that could be removed if using it?